src/pool/srv_pool.c

```diff
@@ -3761,23 +3844,24 @@ ds_pool_update(uuid_t pool_uuid, crt_opcode_t opc,
 	if (rc)
 		D_GOTO(out, rc);
 
-	if (updated && opc == POOL_EXCLUDE) {
-		char	*env;
-		int	 ret;
-
-		env = getenv(REBUILD_ENV);
-		if ((env && !strcasecmp(env, REBUILD_ENV_DISABLED)) ||
-		    daos_fail_check(DAOS_REBUILD_DISABLE)) {
-			D_DEBUG(DB_TRACE, "Rebuild is disabled\n");
-		} else { /* enabled by default */
-			ret = ds_rebuild_schedule(pool_uuid, *map_version,
-						  &target_list);
-			if (ret != 0) {
-				D_ERROR("rebuild fails rc %d\n", ret);
-				if (rc == 0)
-					rc = ret;
-			}
-		}
+	if (!updated || !(opc == POOL_EXCLUDE || opc == POOL_ADD))
+		D_GOTO(out, rc);
+
+	env = getenv(REBUILD_ENV);
+	if ((env && !strcasecmp(env, REBUILD_ENV_DISABLED)) ||
+	    daos_fail_check(DAOS_REBUILD_DISABLE)) {
+		D_DEBUG(DB_TRACE, "Rebuild is disabled\n");
+		D_GOTO(out, rc);
+	}
+
+	int ret;
+	daos_rebuild_opc_t op = (opc == POOL_EXCLUDE ? RB_OP_FAIL : RB_OP_ADD);
```

Comment:
No C99 declare, please declare variable at the begin of code block or function.

Reply:
Fixed.